### PR TITLE
Fixes broken links in the guides

### PR DESCRIPTION
**Pull request by matthewshaver**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/guides/legacy/debugging-schema-names.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can also follow along via this video:
 Do a file search to check if you have a macro named `generate_schema_name` in the `macros` directory of your project.
 
 #### I do not have a macro named `generate_schema_name` in my project
-This means that you are using dbt's default implementation of the macro, as defined [here](https://github.com/dbt-labs/dbt-core/blob/HEAD/core/dbt/include/global_project/macros/etc/get_custom_schema.sql#L17-L30)
+This means that you are using dbt's default implementation of the macro, as defined [here](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/include/global_project/macros/get_custom_name/get_custom_schema.sql#L17-L30)
```
Comment (runleonarun) [inline website/docs/guides/legacy/debugging-schema-names.md:19]: We should come up with a standard here for linking to Core.

File: website/docs/guides/legacy/debugging-errors.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Learning how to debug is a skill, and one that will make you great at your role!
     - The `target/run` directory contains the SQL dbt executes to build your models.
     - The `logs/dbt.log` file contains all the queries that dbt runs, and additional logging. Recent errors will be at the bottom of the file.
     - **dbt Cloud users**: Use the above, or the `Details` tab in the command output.
-    - **dbt CLI users**: Note that your code editor _may_ be hiding these files from the tree <Term id="view" /> ([Atom help](https://discuss.atom.io/t/all-gitignored-files-are-hidden-now-atom-1-15/39238), [VSCode help](https://stackoverflow.com/questions/42891463/how-can-i-show-ignored-files-in-visual-studio-code)).
+    - **dbt CLI users**: Note that your code editor _may_ be hiding these files from the tree <Term id="view" /> [VSCode help](https://stackoverflow.com/questions/42891463/how-can-i-show-ignored-files-in-visual-studio-code)).
```
Comment (runleonarun) [inline website/docs/guides/legacy/debugging-errors.md:14]: I really wish we could link to something other than Stack Overflow here.

**Review by runleonarun — DISMISSED** [review]
These new links look good and work for me. Just a few comments.

File: website/docs/guides/legacy/debugging-errors.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Learning how to debug is a skill, and one that will make you great at your role!
     - The `target/run` directory contains the SQL dbt executes to build your models.
     - The `logs/dbt.log` file contains all the queries that dbt runs, and additional logging. Recent errors will be at the bottom of the file.
     - **dbt Cloud users**: Use the above, or the `Details` tab in the command output.
```
Comment (mirnawong1) [inline website/docs/guides/legacy/debugging-errors.md:13]:
```suggestion
    - **dbt Cloud users**&mdash; Use the above, or the `Details` tab in the command output.
```

File: website/docs/guides/legacy/debugging-errors.md (hunk 2 above)
Comment (mirnawong1) [inline website/docs/guides/legacy/debugging-errors.md:14]:
```suggestion
    - **dbt CLI users**&mdash; Note that your code editor _may_ be hiding these files from the tree <Term id="view" /> [VSCode help](https://stackoverflow.com/questions/42891463/how-can-i-show-ignored-files-in-visual-studio-code)).
```

**Review by mirnawong1 — APPROVED** [review]
@matthewshaver lgtm and one small mdash suggestion!

**Pull request merged**